src/components/add-wallet/AddWalletRow.tsx

```diff
@@ -0,0 +1,106 @@
+import { Box, Stack, Text, useForegroundColor } from '@/design-system';
+import { IS_ANDROID, IS_TEST } from '@/env';
+import styled from '@/styled-thing';
+import { useTheme } from '@/theme';
+import React from 'react';
+import { GradientText, Text as RNText } from '../text';
+import { Icon } from '../icons';
+import ConditionalWrap from 'conditional-wrap';
+import { deviceUtils } from '@/utils';
+import { ButtonPressAnimation } from '../animations';
+
+const RainbowText =
+  IS_ANDROID && IS_TEST
```

Comment:
Do you mean `||` here?

Reply:
i just copy/pasted this from the old component (which i now realize i still need to delete), but maybe I can remove it bc it doesn't make sense / seem necessary

Reply:
should  keep is_testing check since detox gets stuck with gradients

and android check I believe was because this gradient has a more iOS feel to it but imo we shouldn't have shit like this

Reply:
altho on 2nd thought gradient must not be affecting if CI has been passing with this change already, down 2 remove